### PR TITLE
Add string format and concatenation examples in concept exercises #848

### PR DESCRIPTION
**Pull request by aage**

### CODE REVIEW

File: concepts/strings/introduction.md [hunk 1]
```diff
@@ -7,3 +7,13 @@ let fruit = "Apple"
 ```
 
 Strings are manipulated by either calling the string's methods, or using the `String` module's functions. Once a string has been constructed, its value can never change. Any methods/functions that appear to modify a string will actually return a new string.
+
+Strings can be concatenated or formatted a few different ways.
+* You can use the [+ operator](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/strings#string-operators) for concatenation:
```
Comment (ErikSchierboom) [inline concepts/strings/introduction.md:12]:
```suggestion
* You can use the `+` operator for concatenation:
```

File: concepts/strings/introduction.md [hunk 2]
```diff
@@ -7,3 +7,13 @@ let fruit = "Apple"
...
+```fsharp
+let sentence = "Apple" + " is a type of fruit."
+```
+* You can use the [sprintf function](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/plaintext-formatting#checking-of-printf-format-strings) for formatting text, where you have placeholders in a format text (`%s` for a `string`) and provide the values as arguments:
```
Comment (ErikSchierboom) [inline concepts/strings/introduction.md:16]:
```suggestion
* You can use the `sprintf function` for formatting text, where you have placeholders in a format text (`%s` for a `string`) and provide the values as arguments:
```

File: exercises/concept/log-levels/.docs/introduction.md [hunk 3]
```diff
@@ -7,3 +7,13 @@ let fruit = "Apple"
 ```
 
 Strings are manipulated by either calling the string's methods, or using the `String` module's functions. Once a string has been constructed, its value can never change. Any methods/functions that appear to modify a string will actually return a new string.
+
+Strings can be concatenated or formatted a few different ways.
+* You can use the [+ operator](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/strings#string-operators) for concatenation:
```
Comment (ErikSchierboom) [inline exercises/concept/log-levels/.docs/introduction.md:12]:
```suggestion
* You can use the `+` operator for concatenation:
```

File: exercises/concept/log-levels/.docs/introduction.md [hunk 4]
```diff
@@ -7,3 +7,13 @@ let fruit = "Apple"
...
+```fsharp
+let sentence = "Apple" + " is a type of fruit."
+```
+* You can use the [sprintf function](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/plaintext-formatting#checking-of-printf-format-strings) for formatting text, where you have placeholders in a format text (`%s` for a `string`) and provide the values as arguments:
```
Comment (ErikSchierboom) [inline exercises/concept/log-levels/.docs/introduction.md:16]:
```suggestion
* You can use the `sprintf` function for formatting text, where you have placeholders in a format text (`%s` for a `string`) and provide the values as arguments:
```

**Review by ErikSchierboom — COMMENTED** [review]
One tiny nit, which is that the introduction documents don't usually link to external resources.

**Comment by aage** [comment]
Ahh, ok! I've commited your suggestions. Would you like to have these squashed? Or doesn't it make a difference?

**Review by ErikSchierboom — APPROVED** [review]
Great!

**Comment by ErikSchierboom** [comment]
I can squash on merging, so there's no need for you to do that.

**Pull request merged**